KDagal-HTML-and-CSS-Week1/Blog Page.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+	<title>RAW HONEY</title>
+    <link href="css/style.css" rel="stylesheet">
+</head>
+
+<body>
+    <header> <h1>IMPORTANCE OF RAW HONEY</h1> 
+        <img src="Honey.jfif" alt="Raw Honey Picture">
+    </header>
+
+    <main> 
+     <section>
+         <h2>Here are 3 of the Importance of raw honey</h2>
+         <article class="art1">
+             <h3>1. A good source of antioxidants</h3>
+              <p>
+                 Raw honey contains an array of plant chemicals that act as antioxidants. Some types of honey have as many antioxidants as fruits and vegetables.<br/>  
+                 Antioxidants help to protect your body from cell damage due to free radicals. Free radicals contribute to the aging process and may also contribute to the development of chronic diseases such as cancer and heart disease.<br/> 
+                 Research shows that antioxidant compounds in honey called polyphenols may play a role in preventing heart disease.
```

Comment:
This is great info, but if sometimes you need to add text to your website just for the sake of adding some text, you can use [Lorem Ipsum](http://www.lipsum.com/) to generate random paragraphs, could save you some time :)